doc/nrf/libraries/others/identity_key.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_identity_key:
+
+Identity key
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The identity key library manages an asymmetric key used for identity services on devices with the Arm CryptoCell and KMU peripherals.
+
+Functionality
+*************
+
+This library manages the identity keys, which are asymmetric keys intended to provide identity to specific devices.
+The identity key is designed to be unique and is provisioned either device-generated or otherwise in a secure manner during production.
+The identity key is encrypted and stored in the Key Management Unit (KMU), in two reserved KMU slots.
+The HUK used to encrypt the identity key is the Master Key Encryption Key (MKEK), which is a key reserved for encrypting key-wallet keys.
```

Comment:
What is a key-wallet in this context?

Reply:
Would it be equally accurate to say
"a key reserved for encrypting other keys."

?

Reply:
Here, from our docs, this makes more sense:

"Master key encryption key (MKEK). It is used for deriving Key Encryption Keys (KEKs), which are used to encrypt other keys when these are stored."

So we could write

"which is a key used for deriving Key Encryption Keys (KEKs)."

Reply:
Key Encryption Keys are luckily named well enough that they need no explanation.

Reply:
I rephrased it and hopefully now it is more clear to read.

Reply:
Better now :)